src/components/common/TmBalance.vue

```diff
@@ -150,7 +145,7 @@ export default {
         query: UserTransactionAdded,
         result() {
           // query if successful or not as even an unsuccessful tx costs fees
-          this.$apollo.queries.overview.refetch()
+          refetch(this.$apollo.query.overview)
```

Comment:
I would suggest using the Vue Apollo API to just set the options: https://apollo.vuejs.org/api/smart-query.html#setoptions

(If we keep this it would be better to rename it to `refetchNetworkOnly`)

Reply:
I went the same path before. But I think we want to query the cache and only want to force an update manually when we know that it changed, right?

Reply:
Feel free to play with just setting the option though. Wanted to assist you on the problem only.

Reply:
Sure, I guess in the case of balances we would almost always want to query the network though?

Reply:
I would guess almost never. Only if the user does a transaction, right?

Reply:
Rewards: Always.